### DISCUSSION TITLE
It is possible to manual review misidentifications?

### DISCUSSION THREAD

**Discussion by calyptura** [mcguirepr89/BirdNET-Pi#757] [Q&A]
I'm getting a lot of misidentifications. 

Is there any way that I can manually correct misidentified records instead of erasing them?

Thanks for any help.

All the best!

Luciano

**Comment by ehpersonal38** [comment]
In Tools -> Database Maintenance I believe you can multiselect detections and then change the common and scientific name, thus correcting the misidentifications. Be aware it's not the most intuitive tool to use and may require some fiddling around.

**Reply by calyptura** [reply]
Thanks! I just did that. It looked like it worked in the database, but the audio files of all the records I edited disappeared. Picture below. Another question, there isn't a way that at the same time, I can correct the misidentifications and somehow help to improve the model.
![Captura de Tela 2023-02-22 às 23 04 03](https://user-images.githubusercontent.com/126075135/220806110-692c8504-28d9-4f5e-a879-5bc1e08bf410.png)

**Reply by ehpersonal38** [reply]
Ah, I see the issue. So this is going to get a little convoluted but bare with me.

So if you hover over the play button of those detections without spectrograms, a little label will pop up that has the file path. Like this:
![chrome_hN4CXerJy7](https://user-images.githubusercontent.com/103586016/220815192-1453bb7b-a872-4b02-a1a7-93845898106a.png)

In this example I renamed my Yellow-rumped warbler detection to a Giant Antshrike. The problem is that actual file is located in 
`/By_Date/2023-02-20/Yellow-rumped_Warbler/Yellow-rumped_Warbler-55-2023-02-20-birdnet-18:31:59.mp3`
NOT
`/By_Date/2023-02-20/Giant Antshrike/Yellow-rumped_Warbler-55-2023-02-20-birdnet-18:31:59.mp3`
Notice how the common names are mismatched there ^

So what you need to do is navigate to where the file actually lives, and move it to the directory of the new bird name.

So I would move 
`/home/pi/BirdSongs/Extracted/By_Date/2023-02-20/Yellow-rumped_Warbler/Yellow-rumped_Warbler-55-2023-02-20-birdnet-18:31:59.mp3` 
to 
`/home/pi/BirdSongs/Extracted/By_Date/2023-02-20/Giant_Antshrike/Yellow-rumped_Warbler-55-2023-02-20-birdnet-18:31:59.mp3` 

And then the web interface would know be able to find the file again and all would be well. So yeah if you can wrap your head around all that, I wish you the best of luck :)

> Another question, there isn't a way that at the same time, I can correct the misidentifications and somehow help to improve the model.

No there's not and I doubt there ever will be unfortunately. It's something that has been talked about a lot.
